src/colvarcomp_gpath.cpp

```diff
@@ -317,13 +353,13 @@ colvar::gzpath::gzpath(std::string const &conf): CartesianBasedPath(conf) {
         cvm::log(std::string("Geometric path z(σ) will use the square of distance from current frame to path compute z\n"));
     }
     if (total_reference_frames < 2) {
-        cvm::error("Error: you have specified " + cvm::to_str(total_reference_frames) + " reference frames, but gzpath requires at least 2 frames.\n");
-        return;
+        error_code |= cvm::error("Error: you have specified " + cvm::to_str(total_reference_frames) + " reference frames, but gzpath requires at least 2 frames.\n", COLVARS_INPUT_ERROR);
```

Comment:
Why is the return statement moved to the end of the function? I am a bit worried that if it doesn't return here, the following code would segfault or crash, which is OK for NAMD but not a good user experience for VMD.

Reply:
Ok with me to return early, I just overlooked the possibility of a crash.

Reply:
Changed the instances that you pointed out into an early return with error.